index.js

```diff
@@ -132,8 +133,6 @@ exports.checkInvalidCheap = function (state, msg) {
   }
   if(!isValidOrder(msg, true))
     return fatal(new Error('message must have keys in allowed order'))
-  if(!isSupportedHash(msg))
```

Comment:
looks like a regression - if any hash is allowed then upgrading hashes wont work. also, other up to date nodes will reject these messages

Reply:
there should be test coverage for this, though.

Reply:
that check is already in isInvalidShape https://github.com/ssbc/ssb-validate/blob/master/index.js#L83

Reply:
oh, okay great!